reference/docs-conceptual/community/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,112 @@
+  - See style guide for formatting specific content
+- Reorganize examples as follows:
+  - Intro sentence(s)
+  - Code and output
+  - Detailed explanation of code (as necessary)
+- Check hyperlinks for accuracy
+  - Replace or remove TechNet/MSDN links
+  - Ensure minimum number of redirects to target
+  - Ensure HTTPS
+  - Correct link type
+    - File links for local files
+    - URL links for files outside of the docset
+  - Remove locales from URLs
+  - Simplify URLs pointing to `docs.microsoft.com`
+
+## Branch Merge Process
+
+The staging branch is the only branch that should ever be merged into live. Merges from short-lived
+(working) branches should be squashed.
+
+| *Merge from/to*  | *release-branch* | *staging*        | *live*      |
+| ---------------- |:----------------:|:----------------:|:-----------:|
+| *working-branch* | squash and merge | squash and merge | Not allowed |
+| *release-branch* | &mdash;          | merge            | Not allowed |
+| *staging*        | rebase           | &mdash;          | merge       |
+
+### PR Merger checklist
+
+- Content review complete
+- Correct target branch for the change
+- No merge conflicts
+- All validation and build step pass
+  - Warnings and suggestions should be fixed (see [Notes](#notes) for exceptions)
+  - No broken links
+- Merge according to table
+
+### Notes
+
+The following warnings can be ignored:
+
+- > Can't find service name for `<version>/<modulepath>/About/About.md`
+
+- > Metadata with following name(s) are not allowed to be set in Yaml header, or as file level
+  > metadata in docfx.json, or as global metadata in docfx.json: `locale`. They are generated by
+  > Docs platform, so the values set in these 3 places will be ignored. Please remove them from all
+  > 3 places to resolve the warning.
+
+When a PR is merged, the HEAD of the target branch is changed. Any open PRs that were based on the
+previous HEAD are now outdated. The outdated PR can be merged using Admin rights to override the
+merge warnings in GitHub. This is safe to do if the previously merged PR(s) have not touched the
+same files. However, clicking the **Update Branch** button is the safest option. You may have
+unresolved conflicts that need to be fixed.
+
+## Publishing to Live
+
+Periodically, the changes accumulated in the `staging` branch need to be published to the live
+website. This require merging the `staging` branch into the `live` branch.
+
+- The `staging` branch should be merged to `live` at least once per week.
+- The `staging` branch should be merged to `live` after any large accumulation of changes.
+  - Changes to 50 or more files
+  - After merging a release branch
+- The `staging` branch should be merged to `live` after any significant change.
+  - Changes to repo or docset configurations (docfx.json, OPS configs, build scripts, etc.)
+  - Changes to the redirection file
+  - Changes to the TOC
+  - After merging a "project" branch (content reorg, bulk update, etc.)
```

Comment:
This feels partially redundant to the previous line item. Can they be reworded and merged?

Reply:
Collapsed